### PR TITLE
Corrected commit message on PR for issue #4442

### PR DESCRIPTION
**Pull request by C-Lion**

### CODE REVIEW

**Comment by hennevogel** [comment]
@C-Lion are you going to address the feedback you've got?

**Comment by C-Lion** [comment]
Yes I will.

**Comment by C-Lion** [comment]
I made what I think are the requested changes to the commit message and description per the feedback. If not, please clarify further & I'll try again. I am not sure if I should repeat all the information again in the description or not. 

I tried to resubmit the pull request, but it seems not to work that way so I changed the PR title to make it clear that I made the requested change but not sure if I was supposed to do that.

Kindly let me know if I need to something else or change it in some other way to resubmit the PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ The Open Build Service documentation is hosted in a separated repository called
 The Open Build Service is part of the openSUSE project. We follow all the [openSUSE Guiding
 Principles!](http://en.opensuse.org/openSUSE:Guiding_principles) If you think
 someone doesn't do that, please let any of the [openSUSE
-owners](https://github.com/orgs/openSUSE/teams/owners) know!
+owners](https:/en.openSUSE.org/openSUSE:Board) know!
```
Comment (eduardoj) [inline CONTRIBUTING.md]: Just a typo: it should be `https://` instead of `https:/`

**Pull request merged**